Wrapping/Generators/Python/Tests/extras.py

```diff
@@ -493,6 +503,21 @@ def custom_callback(name, progress):
     assert np.allclose(arr.transpose(), itk.array_view_from_image(image))
     assert np.allclose(arr.shape[::-1], itk.array_view_from_image(image).shape)
 
+    data_array = xr.DataArray(arr, dims=['q','x','y'])
```

Comment:
is `q` an error?

Reply:
`q` is an example of an unsupported dimension, added here to check raising an exception with unsupported dimension.